### ISSUE TITLE
Add label to analytics toggle for accessibility

### ISSUE THREAD

**Issue by hspinks** [WorldHealthOrganization/app#926]
Accessibility Audit via @avkvirtru: 

**Problem:** The analytics toggle on the settings screen is not labeled, so VoiceOver cannot tell users what is being toggled.

**Solution:** Add "Analytics" label

**Acceptance Criteria:**
> "Toggle buttons MUST convey an accessible name with programmatically associated, visible labels and convey their state."

---
- [x] Searched the [existing issues](https://github.com/WorldHealthOrganization/app/issues) to ensure you are not creating a duplicate.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).

**Comment by britannio** [comment]
https://github.com/WorldHealthOrganization/app/pull/933#issuecomment-612249878
